### PR TITLE
added bash completion support for katello-service

### PR DESCRIPTION
**Pull request by sean797**
This is the first time I've used spec files so let me know if I haven't quite got it right!

### CODE REVIEW

**Comment by stbenjam** [comment]
[test]

**Comment by stbenjam** [comment]
@sean797 Thanks, this works really nicely!

Spec looks OK to me, although I don't know if we need to put it in the completions/ directory.  We'll see what Jenkins and @ehelms say about it.

**Comment by ehelms** [comment]
I think the sources need to be flat and not nested in a directory.

**Comment by stbenjam** [comment]
Indeed: 

```
    Bad file: /var/lib/workspace/workspace/packaging_build_rpm_katello/rel-eng/build/rpmbuild-katello1pqD3v/SOURCES/katello-git-31.3dd1c2d/katello-service.sh: No such file or directory
```

**Comment by sean797** [comment]
@stbenjam Okay, I've removed the completions directory.

**Comment by stbenjam** [comment]
[test]

File: katello/katello.spec [hunk 1]
```diff
@@ -63,6 +64,10 @@ install -Dp -m0755 %{SOURCE5} %{buildroot}%{_sbindir}/service-wait
 install -Dp -m0755 %{SOURCE2} %{buildroot}%{_bindir}/katello-remove
 install -Dp -m0755 %{SOURCE1} %{buildroot}/usr/share/foreman/script/foreman-debug.d/katello-debug.sh
 
+# install tab completion scripts
+install -d ${buildroot}/etc/bash_completion.d
+install -m 644 %{SOURCE8} ${buildroot}/etc/bash_completion.d/katello-service
```
Comment (stbenjam) [inline katello/katello.spec]: Should be `%{buildroot}` (instead of the $) for both of the lines above

**Comment by sean797** [comment]
Thanks @stbenjam let's try it again!

**Comment by stbenjam** [comment]
[test]

**Comment by stbenjam** [comment]
Oops, thanks for the reminder @ehelms  :-) It looks good to me and works well.

@sean797 One last thing, would you mind squashing everything into one commit?

**Comment by sean797** [comment]
Sure @stbenjam! Created new PR #110

**Comment by stbenjam** [comment]
Thanks! 

You can actually do it in the same PR with a force push, http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html, but new PR works too.

**Comment by sean797** [comment]
Thanks @stbenjam I didn't know that! I appreciate it

**Pull request closed without merging**